src/Common/src/TypeSystem/Common/Utilities/LockFreeReaderHashtable.cs

```diff
@@ -36,11 +36,11 @@ abstract public class LockFreeReaderHashtable<TKey, TValue> where TValue : class
         private static TValue[] s_hashtableInitialArray = new TValue[1];
 
         /// <summary>
-        /// _count represents the current count of elements in the hashtable
-        /// _count is used in combination with _resizeCount to control when the 
+        /// Count represents the current count of elements in the hashtable
```

Comment:
Nit: If you say "Gets the current count...", we won't need to update this if we ever decide to rename it again :)

Does it make sense to refer to _resizeCount from a comment on a public property?